build/repositories/pull-request-repository.js

```diff
@@ -0,0 +1,19 @@
+import { knex } from '../../db/knex-database-connection.js';
+
+async function save({ number, repositoryName }) {
+  return knex('pull_requests').insert({ number, repositoryName });
+}
+
+async function isAtLeastOneMergeInProgress() {
+  const isAtLeastOneMergeInProgress = await knex('pull_requests')
+    .select('isCurrentlyMerge')
+    .where({ isCurrentlyMerge: true })
+    .first();
+  return Boolean(isAtLeastOneMergeInProgress);
```

Comment:
```suggestion
async function isAtLeastOneMergeInProgress() {
  const isAtLeastOneMergeInProgress = await knex('pull_requests')
    .select('isCurrentlyMerged')
    .where({ isCurrentlyMerged: true })
    .first();
  return Boolean(isAtLeastOneMergeInProgress);
```

On devrait peut-être retourner la PR en cours de merge et laisser le soin au code métier d'en faire ce qu'il veut.

Je propose de renommer `isAtLeastOneMergeInProgress` en `getMergeInProgress`